mdoc/Consts.cs

```diff
@@ -4,6 +4,6 @@ namespace Mono.Documentation
 	public static class Consts
 	{
 		// this is only a placeholder
-		public static string MonoVersion = "5.0.0.7";
+		public static string MonoVersion = "5.0.0.8";
```

Comment:
I'd suggest "mimicking" xamarin-android and various other repos:

1. Have a build-time generated file which contains the version number.
2. Version number consists of a hand-curated major.minor.build version, and a generated revision value.
3. The revision value is the number of commits since the last change in the version number.

For example, in xamarin-android the [`$(ProductVersion)` MSBuild property](https://github.com/xamarin/xamarin-android/blob/master/Configuration.props#L15) contains the major.minor.build value, and the revision value [is computed](https://github.com/xamarin/xamarin-android/blob/master/build-tools/scripts/BuildEverything.mk#L7-L11) via `git log <version-change-commit>..HEAD --oneline | wc -l`.

This is particularly handy when you have a CI system building all commits, as every build will have a different, unique, version.